.github/workflows/verify.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches: [ master ]
```

Comment:
Why only on master?

Reply:
Followed GitHub's template for Java. I'll change it.